src/meta/buildTags.jsx

```diff
@@ -62,6 +62,84 @@ const buildTags = (config) => {
 
     if (config.openGraph.type) {
       tagsToRender.push(<meta key="og:type" property="og:type" content={config.openGraph.type} />);
+
+      if (config.openGraph.type == 'profile' && config.openGraph.profile) {
+        if (config.openGraph.profile.first_name) {
```

Comment:
Could you switch out all of the `snake_case` to be `camelCase`. 

I made a mistake add `site_name` as `snake_case` 🙈